tests/smoke_tests/test_examples.py

```diff
@@ -120,3 +120,54 @@ def test_ray_train(generic_cloud: str, accelerator: Dict[str, str]) -> None:
             timeout=20 * 60,
         )
         smoke_tests_utils.run_one_test(test)
+
+
+# ---------- Test NeMo RL ----------
+@pytest.mark.no_scp  # SCP does not support num_nodes > 1 yet
+@pytest.mark.no_hyperbolic  # Hyperbolic not support num_nodes > 1 yet
+@pytest.mark.no_seeweb  # Seeweb does not support multi-node
+@pytest.mark.no_kubernetes  # No enough resources in Kubernetes yet
+@pytest.mark.resource_heavy
+@pytest.mark.parametrize('accelerator', [{
+    'do': 'H100',
+    'nebius': 'L40S',
+}])
+def test_nemorl(generic_cloud: str, accelerator: Dict[str, str]) -> None:
+    cpu = '10+'
+    memory = '60+'
+    if generic_cloud == 'kubernetes':
```

Comment:
Keep this block here so that when we support Kubernetes later, we can simply remove L129.